pyproject.toml

```diff
@@ -1,22 +1,239 @@
-[tool.black]
-target-version = ["py36"]
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
+build-backend = "setuptools.build_meta"
```

Comment:
i prefer hatchling or something similar as a backend but i suppose setuptools is fine as a first pass